Update schedule (#510)
carpentries/lesson-development-training: instructors/instructor-notes.md

There is an extended break for lesson trial runs between parts 1 & 2, where learners are expected to conduct mini pilots of their lessons with the 
intended target audiences.

> Below is a suggested schedule that can be used as a guideline to plan the content and timings for each teaching session. You should plan to take a break after

Author: ```suggestion
Below is a suggested schedule that can be used as a guideline to plan the content. The timings for each teaching episode vary depending on the audience and the pace so we try not to give suggestions any more as 
we are likely to be wrong. You should plan to take a break after
```

> approximately one hour of teaching.

Reviewer: ```suggestion
every one hour of teaching, approximately.
```

### Part 1

#### Day 1 (09:00 - 13:00)

- Introduction
- Lesson Design
- Identifying Your Target Audience
- Defining Lesson Objectives

> #### Day 2 (09:00 - 13:00)

Reviewer: ```suggestion
#### Day 2
```
matching last suggestion
